tests/test_generate.py

```diff
@@ -47,17 +55,30 @@ def test_generate(yaml_dir, tmpdir):
         project=parsed_credentials.project_id, credentials=parsed_credentials
     )
 
-    # Clean up any previous test data.
+    return test_bucket, credentials, storage_client
+
+
+# Fetches the pages used for testing
+def fetch_pagedata(storage_client, test_bucket):
```

Comment:
Perhaps name this `setup_tesdata`? Not sure the best name, but `pagedata` doesn't seem accurate.

Reply:
`testdata` sounds more inclusive for the variety here, I'll go ahead with that